doc/invalidation-handlers.rst

```diff
@@ -67,12 +67,17 @@ Assume you sent four responses:
 
 You can now invalidate some URLs using tags::
 
-    $tagHandler->invalidateTags(array('group-a', 'tag-four'))->flush();
+    $tagHandler->invalidateTags(array('group-a', 'tag-four'));
 
 This will ban all requests having either the tag ``group-a`` /or/ ``tag-four``.
 In the above example, this will invalidate ``/two``, ``/three`` and ``/four``.
 Only ``/one`` will stay in the cache.
 
+.. note::
+
+    The ``invalidateTags()`` method talks directly with the cache proxy for an
+    immediate affect. There is not need to call ``flush()``.
```

Comment:
this is a mistunderstanding. the tag handler talks with the CacheInvalidator. so you do need to call $cacheInvalidator->flush() at some point. you can link to the corresponding section in cache-invalidator with ``:ref:`flush```.